dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/cards/conditional-formatting.md

```diff
@@ -33,7 +59,9 @@ You can create format rules in the following ways:
 
     ![AddFormatRule_ValueItem](../../../../images/addformatrule_valueitem118549.png)
 
-## Appearance Settings   
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting Common](../../appearance-customization/conditional-formatting.md).    
```

Comment:
Is the title of the topic "Conditional Formatting" (not "Conditional Formatting Common")?

Reply:
I see. I think it might be better to say "Conditional Formatting", if it's possible to change it.